source/Public/SQL Database/New-FabricSQLDatabase.ps1

```diff
@@ -42,7 +43,10 @@ function New-FabricSQLDatabase
 
         [Parameter(Mandatory = $false)]
         [ValidateNotNullOrEmpty()]
-        [string]$Description
+        [string]$Description,
+
+        [Parameter(Mandatory = $false)]
```

Comment:
Possibly one to add to the standard but this is not needed, parameters are not mandatory by default in PowerShell.

```suggestion
```